config/package/usr/lib/systemd/system/socorro-middleware.service

```diff
@@ -3,8 +3,8 @@ Description=Socorro Middleware
 
 [Service]
 Environment=VENV=/data/socorro/socorro-virtualenv
-ExecStart=/bin/bash -c "envconsul -upcase=false -prefix socorro/common -prefix socorro/middleware $VENV/bin/uwsgi -H $VENV -M --need-app -w wsgi.middleware -s /var/run/uwsgi/socorro/socorro-middleware.sock --chmod-socket=664 --uid=socorro --gid=nginx"
-Restart=always
+ExecStart=/bin/bash -c "echo No more middleware"
+Restart=no
```

Comment:
This makes sure that when the process ends, it doesn't get restarted. Thus systemd will bring this service up, it echos a line, it ends, and then systemd is all like, "well, fine. be that way."